lib/cli.rb

```diff
@@ -404,6 +404,23 @@ def self.check_port_validity(port)
     end
   end
 
+  desc "Move system description"
+  long_desc <<-LONGDESC
+    Move a system description.
+
+    The system description is moved under the provided name.
+  LONGDESC
+  arg "FROM_NAME"
+  arg "TO_NAME"
+  command "move" do |c|
+    c.action do |global_options,options,args|
```

Comment:
Unused block argument - `global_options`. If it's necessary, use `_` or `_global_options` as an argument name to indicate that it won't be used.<br>Unused block argument - `options`. If it's necessary, use `_` or `_options` as an argument name to indicate that it won't be used.<br>Space missing after comma.